FAIL: vsnprintf-chk.c execution, -O2 -flto -fno-use-linker-plugin -flto-partition=none

 http://gcc.gnu.org/bugzilla/show_bug.cgi?id=62032

--- gcc-4_9-branch/gcc/lto/ChangeLog
+++ gcc-4_9-branch/gcc/lto/ChangeLog
@@ -1,3 +1,11 @@
+2015-06-30  Bin Cheng  <bin.cheng@arm.com>
+
+	Backport from mainline
+	2014-08-08  Bin Cheng  <bin.cheng@arm.com>
+
+	PR lto/62032
+	* lto-lang.c (lto_init): Switch mis-matched arguments.
+
 2015-06-23  Release Manager
 
 	* GCC 4.8.5 released.
--- gcc-4_9-branch/gcc/lto/lto-lang.c	2014/08/15 02:52:55	213999
+++ gcc-4_9-branch/gcc/lto/lto-lang.c	2014/08/15 05:49:56	214000
@@ -1185,8 +1185,8 @@ lto_init (void)
     }
   else
     {
-      lto_define_builtins (va_list_type_node,
-			   build_reference_type (va_list_type_node));
+      lto_define_builtins (build_reference_type (va_list_type_node),
+			   va_list_type_node);
     }
 
   targetm.init_builtins ();
